.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
I like this idea, can we retitle it to 'Support Question'? I definitely want to make it very apparent that support provided via GitHub can be limited and we recommended Discord for both assistance in using CMaNGOS as well as for development related questions.

We might even add that if it's a support or issue with using CMaNGOS that they provide pretty much the same information as in the bug report (e.g., version informations, how too reproduce, what they've done to troubleshoot the issue thus far, etc...).

Reply:
https://github.com/ulfgebhardt/issues/issues/8